### PR TITLE
[qualys_vmdr] Rename and convert fields. Lower case `cloud.provider`

### PR DESCRIPTION
**Pull request by clement-fouque**
## Proposed commit message

This PR is doing several things: 

### Lower case `cloud.provider`  and `qualys_vmdr.asset_host_detection.cloud_provider`
ECS field `cloud.provider` provides examples in lowercase. As most elastic field values are lowercase, I think it makes sense to align the Qualys integration. I don't want to have to specify query `cloud.provider: "AWS" or cloud.provider: "aws"`

### Rename fields
| Previous field name | New field name |
|--------|--------|
| qualys_vmdr.asset_host_detection.vulnerability.qds.text | qualys_vmdr.asset_host_detection.vulnerability.qds.score |
| qualys_vmdr.asset_host_detection.affect.exploitable_config | qualys_vmdr.asset_host_detection. affect_exploitable_config |
| qualys_vmdr.asset_host_detection.affect.running.kernel | qualys_vmdr.asset_host_detection.affect_running_kernel |
| qualys_vmdr.asset_host_detection.affect.running.service | qualys_vmdr.asset_host_detection.affect_running_service |
| qualys_vmdr.asset_host_detection.first.found_datetime | qualys_vmdr.asset_host_detection.first_found_datetime | 
| qualys_vmdr.asset_host_detection.first.reopened_datetime | qualys_vmdr.asset_host_detection.first_reopened_datetime | 
| qualys_vmdr.asset_host_detection.last.fixed_datetime | qualys_vmdr.asset_host_detection.last_fixed_datetime | 
| qualys_vmdr.asset_host_detection.last.found_datetime | qualys_vmdr.asset_host_detection.last_found_datetime | 
| qualys_vmdr.asset_host_detection.last.processed_datetime | qualys_vmdr.asset_host_detection.last_processed_datetime | 
| qualys_vmdr.asset_host_detection.last.reopened_datetime | qualys_vmdr.asset_host_detection.last_reopened_datetime | 
| qualys_vmdr.asset_host_detection.last.test_datetime | qualys_vmdr.asset_host_detection.last_test_datetime | 
| qualys_vmdr.asset_host_detection.last.update_datetime | qualys_vmdr.asset_host_detection.last_update_datetime | 
| qualys_vmdr.asset_host_detection.times.found | qualys_vmdr.asset_host_detection.times_found | 
| qualys_vmdr.asset_host_detection.times.reopened | qualys_vmdr.asset_host_detection.times_reopened | 

### Convert field types

| Field | Old type | New type |
|--------|--------|--------|
| qualys_vmdr.asset_host_detection.asset_id | keyword | long |
| qualys_vmdr.asset_host_detection.vulnerability.qds.score (initially text) | keyword | integer |
| qualys_vmdr.asset_host_detection.vulnerability.qid | keyword | integer |

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/qualys_vmdr/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "5.0.0"
+  changes:
+    - description: Rename fields to match Qualys name. Convert fields to long/integer. Lowercase `cloud.provider` field
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/10966
```
Comment (efd6) [inline packages/qualys_vmdr/changelog.yml:6]:
```suggestion
    - description: Rename fields to match Qualys name.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/10966
    - description: Convert numeric fields to long/integer.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/10966
    - description: Lowercase `cloud.provider` field.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/10966
```

File: packages/qualys_vmdr/data_stream/asset_host_detection/sample_event.json
Comment (efd6) [inline packages/qualys_vmdr/data_stream/asset_host_detection/sample_event.json:1]: The sample event here does not look like it was produced by a system test; the fields are out of order. Can you fix this please?

Comment (clement-fouque) [reply packages/qualys_vmdr/data_stream/asset_host_detection/sample_event.json:1]: I didn't know we could generate it automatically! I was updating it manually or taking it from "production" directly. 

If anyone else is reading, here is the command: `elastic-package test system --generate`

Comment (efd6) [reply packages/qualys_vmdr/data_stream/asset_host_detection/sample_event.json:1]: Have you pushed the update?

**Review by efd6 — DISMISSED** [review]
Thanks

**Comment by clement-fouque** [comment]
@efd6 I pushed the last update (generated sample events from system test). Once the build will be successful, it's ready to be approved and hopefully merged.

**Comment by efd6** [comment]
Probably premature, this will fail due to README.md mismatch. Please also `elastic-package build`.

**Comment by clement-fouque** [comment]
Learning is continuing 😉!

**Review by efd6 — APPROVED** [review]

**Pull request merged**